averages.js

```diff
@@ -0,0 +1,95 @@
+module.exports = {
+
+  total: function(array){
```

Comment:
the goal was to add these to the Array prototype - which i see you have commented out below. were you not able to get it to work that way?